portfolio/src/main/webapp/data.js

```diff
@@ -0,0 +1,4 @@
+// arrays store text that will be displayed on pages 
+var moments = [
+    
+]
```

Comment:
This file needs a newline at the end (same with other files)

Reply:
Nick gave me comments to not have a new line at the end - what should I do?

Reply:
Oops. Each file should end with a new line.